src/vl-map-layer.js

```diff
@@ -131,16 +150,22 @@ export class VlMapLayer extends vlElement(HTMLElement) {
     return this.getAttribute('max-resolution') || Infinity;
   }
 
-  get _map() {
-    return this._mapElement.map;
-  }
-
-  get _mapReady() {
-    return this._mapElement.ready;
+  get _mapElement() {
+    if (this.parentNode && this.parentNode.map) {
+      return this.parentNode;
+    } else {
+      return null;
+    }
   }
 
-  get _mapElement() {
-    return this.parentNode;
+  /**
+   * Geeft de waarde op basis van een sleutel.
+   *
+   * @param {String} key
+   * @return {Object}
+   */
+  get(key) {
+    return this._layer.get(key);
```

Comment:
ik zou vermijden om dit in de API te steken, en de boel een beetje meer te encapsuleren?
voorlopig alleen getTitle() nodig?

Reply:
Ik zal nog een `getTitle` toevoegen, maar ik denk dat een generieke get sowieso interessant kan zijn om in de toekomst niet voortdurend releases te moeten voorzien voor elke getter die applicatief gebruikt moet worden en nog niet voorzien werd via een wrapper.